bolt-sidecar/.env.example

```diff
@@ -33,8 +33,8 @@ BOLT_SIDECAR_BUILDER_PRIVATE_KEY=
 BOLT_SIDECAR_MAX_COMMITMENTS_PER_SLOT=128
 # Max committed gas per slot
 BOLT_SIDECAR_MAX_COMMITTED_GAS_PER_SLOT=10_000_000
-# Min priority fee to accept for a commitment
-BOLT_SIDECAR_MIN_PRIORITY_FEE=2000000000 # 2 Gwei = 2 * 10^9 wei
+# Min profit per gas to accept a commitment
+BOLT_SIDECAR_MIN_PROFIT=2000000000 # 2 Gwei = 2 * 10^9 wei
```

Comment:
This needs to be calculated and based on profit a proposer needs to exceed the "risk free" return of liquid staking on the collateral